modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -519,9 +519,9 @@ const Options = ({
 		];
 	};
 
-	const composedAdd = compose(clone, dedup, add, set);
+	const composedAdd = compose(clone, add, normalize, set);
```

Comment:
Hey @cgmonte, these values can be sorted?

Reply:
No, the order matter here. The return of `clone` is passed to `dedup`, the return of `dedup` is passed to `add` and so on. This is what `compose` does.